podcasts/End of Year/Stories/TopFivePodcastsStory.swift

```diff
@@ -28,12 +28,14 @@ struct TopFivePodcastsStory: ShareableStory {
                 .padding([.leading, .trailing], 35)
                 .padding(.top, headerSpacing)
             }.background(
-                ZStack(alignment: .bottom) {
+                ZStack(alignment: .top) {
                     Color.black
 
                     StoryGradient()
-                    .offset(x: geometry.size.width * 0.7, y: -geometry.size.height * 0.7)
+                    .offset(x: geometry.size.width * 0.7, y: -geometry.size.height * 0.22)
+                    .clipped()
                 }
+                .ignoresSafeArea()
```

Comment:
The problem is that this affects the bigger screens on the opposite way:

| Before the change | After the change |
| ------------------ | ----------------- |
| ![Simulator Screenshot - iPhone 15 - 2023-10-19 at 11 10 58](https://github.com/Automattic/pocket-casts-ios/assets/7040243/6179c883-6d27-4590-a5ca-af629bd21585) | ![Simulator Screenshot - iPhone 15 - 2023-10-19 at 11 10 15](https://github.com/Automattic/pocket-casts-ios/assets/7040243/35850612-a2e1-4103-991b-befc2cad498a) |

Since this also touches on some other stories, would you be ok if I create an issue and tackle that later? I feel it would be easier to test all stories at once.